### DISCUSSION TITLE
Native M1 version

### DISCUSSION THREAD

**Discussion by thedarthmail** [voxeltycoon/issues#954] [General] [answered]
Congratulation on the latest update! With the switch to Unity 2020.3.30, would a native M1 version be possible?

**Comment by andrewpey** [comment]
Thanks for opening this disccussion! We still need to investigate if it's possible for us to add a native M1 version. We'll update this discussion when we have more info.

**Comment by andrewpey** [comment, marked as answer]
We've just published `0.87.1.8` update to the [beta branch](https://voxeltycoon.xyz/devlog/beta-branch/) which includes M1 native build. Please let us know if it has memory problems or if it works at all!

**Comment by thedarthmail** [comment]
Excellent!!! Appears to work fine! Thank you!!!
